deploy/olm-catalog/ocs-operator/0.0.1/cephcluster.crd.yaml

```diff
@@ -108,6 +108,8 @@ spec:
               properties:
                 workers:
                   type: integer
+            removeOSDsIfOutAndSafeToRemove:
```

Comment:
not directly related to this PR, but this magically shows up because we're using rook-ceph master floating tag now. I don't feel great about what's going on here.

basically the image we're building the CSV with is a floating tag, which could change after deployment  making the CSV we generated inaccurate.